src/web/Link.tsx

```diff
@@ -62,9 +66,22 @@ export class Link extends React.Component<Types.LinkProps, {}> {
                 onMouseEnter={ this.props.onHoverStart }
                 onMouseLeave={ this.props.onHoverEnd }
                 onMouseDown={ this._onMouseDown }
+                tabIndex={ this.props.tabIndex }
             >
                 { this.props.children }
             </a>
+        ) : (
+            <button // When url is not set <a> will be unaccessible, so we make it a button.
```

Comment:
This imposes a new limitation on the use of an RX.Link. It can no longer be used inside of an RX.Button because of the nesting rules of <buttons> in HTML. That's probably OK, but it's something we should be aware of.

Reply:
As for this part, the spec says that both `<a>` and `<button>` can't have interactive content:

https://w3c.github.io/html/sec-forms.html#the-button-element
https://w3c.github.io/html/textlevel-semantics.html#the-a-element

It's not that we check for a validity given the dynamic DOM generation, though...